dash/utils/src/__tests__/phone-numbers.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { test, describe, expect } from 'vitest';
+import { parseE164, prettyE164 } from '../phone-numbers';
+
+describe(`parseE164()`, () => {
+  const cases: Array<[string | undefined, string | null]> = [
+    [`+1 (555) 555-5555`, `+15555555555`],
+    [`+1 (555)     555 (5555)`, `+15555555555`],
+    [`(555) 555-5555`, `+15555555555`],
+    [`5555555555`, `+15555555555`],
+    [`+35555555555`, `+35555555555`],
+    [`+35555555555  `, `+35555555555`],
+    [`   +35555555555`, `+35555555555`],
+    [` \t  +35555555555`, `+35555555555`],
+    [`1234567`, `+1234567`], // shortest
+    [`123456`, null],
+    [`+123456`, null],
+    [`123456789012345`, `+123456789012345`], // longest
+    [`+123456789012345`, `+123456789012345`],
+    [`+123(456)78-9-012345`, `+123456789012345`],
+    [`1234567890123456`, null], // too long
+    [`+1234567890123456`, null],
+    [undefined, null],
+  ];
+  test.each(cases)(`%s to I.164 -> %s`, (input, expected) => {
+    expect(parseE164(input)).toBe(expected);
+  });
+});
+
+describe(`prettyE164()`, () => {
```

Comment:
for US numbers, we format them pretty, like `(555) 123-4567`. international numbers we keep in E164 format, i'm guessing most non US folks are used to seeing that